sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -135,6 +133,42 @@ For more information on the OpenTelemetry project, please review the [OpenTeleme
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Program.cs) for a complete demo.
 
+### Log Scopes
+
+Log scopes allow you to add additional properties to the logs generated by your application.
+Although the Azure Monitor Exporter does support scopes, this feature is off by default in OpenTelemetry.
+To leverage log scopes, you must explicitly enable them.
+
+To include the scope with your logs, set `OpenTelemetryLoggerOptions.IncludeScopes` to `true` in your application's configuration:
+```csharp
+var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(options =>
+    {
+        options.AddAzureMonitorLogExporter(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000");
+        options.IncludeScopes = true;
+    });
+});
+```
+
+Once enabled, you can define a scope using a dictionary or any enumerable of KeyValuePair<string, object>.
+All logs written within the context of the scope will include the specified information.
+Azure Monitor will add these scope values to the Log's CustomProperties.
+```csharp
+List<KeyValuePair<string, object>> scope =
+[
+    new("scopeKey", "scopeValue")
+];
+
+using (logger.BeginScope(scope))
+{
+    logger.LogInformation("Example message.");
+}
+```
+
+In scenarios involving multiple scopes or a single scope with multiple key-value pairs, if duplicate keys are present, only the first occurrence of the key-value pair from the outermost scope will be recorded.
```

Comment:
this is better, but it is still missing the priority ordering for the duplicate key within the same scope, as shown below. 
```
List<KeyValuePair<string, object>> scope =
[
    new("scopeKey", "scopeValue1")
    new("scopeKey", "scopeValue2")
];

using (logger.BeginScope(scope))
```

OTel SDK upstream does not de-duplicate, so its is upto Exporters, so I'd suggest to cover this with good unit test coverage as well.

Reply:
> it is still missing the priority ordering for the duplicate key within the same scope

I thought I covered that case with this quote: `or a single scope with multiple key-value pairs`
Do you have any suggestions on how to rephrase this to make it easier to understand?



> I'd suggest to cover this with good unit test coverage

I already added the code sample you shared. 
I can add more tests specifically around scopes.

https://github.com/Azure/azure-sdk-for-net/blob/3f1749c5734a9ac0435790201aef312aea528f05/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/E2ETelemetryItemValidation/LogsTests.cs#L69-L83

Reply:
I found several tests around scopes in the LogsHelperTests class.
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

I added one more to check behavior when the same key is present in both the scope and the log template.